<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HA1101901716" name="US-7.0.0.0.0.12-e55-01_OVF10.ova" tests="8" failures="1" timestamp="9 Nov 2015 06:36:13 GMT" time="2805.871" errors="0">
  <testcase name="AddVMSuite" time="1054.893" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="EditVM" time="42.404" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="StopVM" time="105.819" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="StartVM" time="135.931" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="RefreshVM" time="71.941" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew">
    <failure type="com.avaya.sdmclient.extraResources.MyException" message="Test case failed .. Check screenshot for the same.">
      <![CDATA[com.avaya.sdmclient.extraResources.MyException: Test case failed .. Check screenshot for the same.
	at com.avaya.sdmclient.Settings.checkSuccessOrFailure(Settings.java:1598)
	at com.avaya.sdmclient.Settings.checkSuccessOrFailure(Settings.java:1560)
	at com.avaya.sdmclient.Settings.checkSuccessOrFailure(Settings.java:1586)
	at com.avaya.sdmclient.Settings.checkSuccessOrFailure(Settings.java:1560)
	at com.avaya.sdmclient.runnerdemo.SecondThreadNew.RefreshVM(SecondThreadNew.java:305)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:648)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:834)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1172)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at com.avaya.sdmclient.runnerdemo.settingsForConcThreads.runThread(settingsForConcThreads.java:306)
	at com.avaya.sdmclient.runnerdemo.SecondThreadNew$1MyRunnable.run(SecondThreadNew.java:418)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- RefreshVM -->
  <testcase name="RestartVM" time="138.125" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="DeleteVM" time="40.697" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="startNewThread" time="1216.058" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
</testsuite> <!-- US-7.0.0.0.0.12-e55-01_OVF10.ova -->
